modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -27,14 +27,14 @@ const defaultProps = {
   // Use load directly so we don't use ResourceManager
   fetch: {
     type: 'function',
-    value: (url, {layer, signal}) => {
-      const loadOptions = {...layer.getLoadOptions()};
+    value: (url, {layer, loaders, options, signal}) => {
+      const loadOptions = {...options, ...layer.getLoadOptions()};
       loadOptions.fetch = {
         ...loadOptions.fetch,
         signal
       };
 
-      return load(url, loadOptions);
+      return load(url, loaders, loadOptions);
```

Comment:
I don't think this works if `loaders` is undefined? Have you tested the terrain example?

Reply:
You're right, I misread how the options parsing works. Fixed